### DISCUSSION TITLE
WakaTime user profile not public, but it's public

### DISCUSSION THREAD

**Discussion by gengjiawen** [anuraghazra/github-readme-stats#3690] [Q&A] [answered]
### Describe the bug

https://github-readme-stats.vercel.app/api/wakatime?username=gengjiawen&layout=compact

My profile is public: https://wakatime.com/@gengjiawen

**Comment by qwerty541** [comment, marked as answer]
Hey, @gengjiawen! Our documentation says that to get your card working you need enable BOTH `Display code time publicly` and `Display languages, editors, os, categories publicly` settings. Make sure that your current wakatime settings match this requirement. This works perfectly with my own card.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/4ba08916-8fcc-43b0-a4dd-bc37151dc56d)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/58f87797-1eaa-4a95-acf1-1ed33b247591)

<img src="https://github-readme-stats-steel-omega.vercel.app/api/wakatime?username=qwerty541&show_icons=true&layout=compact&icon_color=2d77dc&title_color=2d77dc&text_color=ffffff&bg_color=0d1117&hide_border=true" />
